component/core/src/main/java/org/exoplatform/social/core/manager/IdentityManagerImpl.java

```diff
@@ -273,7 +273,7 @@ private void detectChanges(Profile specificProfile) {
       if (specificProfile.getProperty(Profile.AVATAR) != null){
         list.add(UpdateType.AVATAR);
       }
-      if (UserProfileComparator.hasChanged(specificProfile, existingProfile, Profile.LAST_LOGIN_TIME)){
+      if (UserProfileComparator.hasChanged(specificProfile, existingProfile, Profile.LAST_LOGIN_TIME) || UserProfileComparator.hasChanged(specificProfile, existingProfile, Profile.ENROLMENT_DATE)){
```

Comment:
in if for CONTACT, remove Profile.LAST_LOGIN_TIME

```suggestion
      if (UserProfileComparator.hasChanged(specificProfile, existingProfile, Profile.LAST_LOGIN_TIME, Profile.ENROLMENT_DATE)){
```